apps/omg_child_chain/lib/omg_child_chain/release_tasks/set_fee_file_adapter_opts.ex

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019-2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.ChildChain.ReleaseTasks.SetFeeFileAdapterOpts do
+  @moduledoc """
+  Detects if `FEE_ADAPTER` is set to `"FILE"` (case-insensitive). If so, it sets the system's
+  fee adapter to FileAdapter and configures it with values from related environment variables.
+  """
+  @behaviour Config.Provider
+  require Logger
+
+  @app :omg_child_chain
+  @config_key :fee_adapter
+  @env_fee_adapter "FEE_ADAPTER"
+  @env_fee_specs_file_path "FEE_SPECS_FILE_PATH"
+
+  def init(args) do
+    args
+  end
+
+  def load(config, _args) do
+    _ = on_load()
+
+    @env_fee_adapter
+    |> System.get_env()
+    |> parse_adapter_value()
+    |> case do
+      "FILE" -> configure_file_adapter(config)
+      _ -> config
```

Comment:
same as above, do we not need to `Config.Reader.merge/2`?

Reply:
In this case no config is changed so I'm passing the original `config` back. Refactored in https://github.com/omisego/elixir-omg/pull/1385/commits/d2b16cad7cc7da18d12d703af6f4a68c64c375db to make this clearer though.